src/api/weather.js

```diff
@@ -0,0 +1,7 @@
+export async function getWeather() {
+  const response = await fetch(
+    `https://api.openweathermap.org/data/2.5/weather?lat=7&lon=51&appid=b12e52770a4ea6451e533761faa2e8ff`
+  );
+  const data = await response.json();
+  return data.results;
```

Comment:
I don't think that there is a `results`property in the response of your api call. Try to log data to understand the structure.

Reply:
I solved the structure problem, thank you for the hint.